### PR TITLE
adding assignment 1 jupyter notebook

### PR DESCRIPTION
**Pull request by maziarf**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
The assignment was about training and testing a KNN classification model. Here, I'm adding responses to the assignment 1 jupyter notebook.
## What did you learn from the changes you have made?
I learned about implementing knn classification, testing and training data splitting, and testing the accuracy of the model. 
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
not really
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
Most of the questions were very similar to the live codes and as a result providing answers didn't pose too much challenge. 
## How were these changes tested?
By executing the code on the notebook.
## Checklist
- [check ] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by KasraVakiloroayaei — CHANGES_REQUESTED** [review]
A1: Incomplete

Please note the following changes:

It is valuable to add a detailed PR description, as it can help you in the future when recalling this information as a resource.

Q2part(i) Standardizing is important, in particular when using a modeling method that takes into consideration a distance calculation, such as KNN. This will ensure that they are all on a similar scale thus, not having one variable over contributing to the distance calculation 
Q2part(ii) More Specifically, since it is a categorical variable, we do not standardize it 
Q2part(iv) An exercise to try is adding a stratify argument in the train_test_split function, and see if that influences the choice for a best hyperparameter k. 
Q3: The parameter grid range was (1,50). Also, the third argument in the range function is not necessary in this instance. 
With the changes above you will get a different value for your hyperparameter, k which the best nearest neighbor value

**Review by FanWang0216 — APPROVED** [review]
